### PR TITLE
Uniformify tests structure

### PR DESCRIPTION
**Pull request by bradleyayers**
A few little touch-ups to make the way tests work in the repo more consistent across packages. See the individual commits for more detail.

### CODE REVIEW

File: packages/cli/src/generator.test.ts [hunk 1]
```diff
@@ -139,7 +139,26 @@ export interface IGetNotificationsQuery {
         types,
         {} as ParsedConfig,
       );
-      expect(result).toMatchSnapshot();
```
Comment (adelsz) [inline packages/cli/src/generator.test.ts:142]: I would prefer a snapshot instead, any reasons why a string would be better?

File: packages/example/package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "lib"
   ],
   "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1",
+    "test": "docker-compose run test",
```
Comment (adelsz) [inline packages/example/package.json:17]: 👍

**Review by adelsz — APPROVED** [review]
Thanks @bradleyayers, looks good to me. Added a comment on the snapshot change.

File: packages/cli/src/generator.test.ts (hunk 1 above)
Comment (bradleyayers) [reply packages/cli/src/generator.test.ts:142]: It simply matches the style of the other tests, they could all be snapshots or all be inline strings, or even be inline snapshots. This change was just one for consistency to have less surprises due to differing conventions when reading the code.

To be honest I'd prefer if they were all snapshots (easier to update if they change).

**Comment by bradleyayers** [comment]
Apologies I only just saw your reply.

File: packages/cli/src/generator.test.ts (hunk 1 above)
Comment (adelsz) [reply packages/cli/src/generator.test.ts:142]: Makes sense, lets migrate them all to snapshots later.

**Pull request merged**